### PR TITLE
feat: add product listing page

### PR DESCRIPTION
**Pull request by devdeadviz**
# Change Logs
## Screenshots
Screenshot (213)
Screenshot (214)

### CODE REVIEW

File: src/pages/ProductsListing/ProductsListing.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import { Filter, Footer, Navbar, ProductsCard } from "../../components";
+import { useProducts } from "../../contexts/products-context";
+import "./ProductsListing.css";
+
+const ProductsListing = () => {
+  const { products: availableProducts } = useProducts();
+
+  return (
+    <>
+      <Navbar />
```
Comment (ivinayakg) [inline src/pages/ProductsListing/ProductsListing.js:10]: there is no need to import the navbar, footer on every page wrap every page in them by using a single file which will handle routes and then there use the navbar and footer

File: src/components/index.js [hunk 2]
```diff
@@ -2,3 +2,5 @@ export { Navbar } from "./Navbar/Navbar";
 export { CategoriesCard } from "./CategoriesCard/CategoriesCard";
```
Comment (ivinayakg) [inline src/components/index.js:2]: in every folder name the js file "index.js" so that it will auto-import that file only from the folder

**Review by ivinayakg — COMMENTED** [review]
I liked the fact that you kept the PR simple and clean😃

File: src/components/index.js (hunk 2 above)
Comment (devdeadviz) [reply src/components/index.js:2]: You are in that file only.

File: src/pages/ProductsListing/ProductsListing.js (hunk 1 above)
Comment (devdeadviz) [reply src/pages/ProductsListing/ProductsListing.js:10]: Okay, I will update this.

File: src/components/index.js [hunk 3]
```diff
@@ -2,3 +2,5 @@ export { Navbar } from "./Navbar/Navbar";
...
 export { UpcomingsCard } from "./UpcomingsCard/UpcomingsCard";
 export { Footer } from "./Footer/Footer";
+export { Filter } from "./Filter/Filter";
```
Comment (yashpatel1797) [inline src/components/index.js:5]: nice that you have created index.js for all imports.

**Review by yashpatel1797 — APPROVED** [review]
1. File structure is well maintained.
2. design looks clean.
3. readability of code is also good.

File: src/components/index.js (hunk 3 above)
Comment (devdeadviz) [reply src/components/index.js:5]: Thank you

**Pull request merged**